### ISSUE TITLE
Organization-level assignment in the permissions modal for the teams tab doesn't work when assigning admin permissions

### ISSUE THREAD

**Issue by dsesami** [ansible/awx#2563]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - UI

##### SUMMARY
Organization-level permissions on the teams page doesn't work. Try to add an organization-wide permission, nothing happens. A POST to `api/v2/teams/1/roles` was observed with the payload:
```{id: 62}```
returning a status code of 400. Nothing else happens.
##### ENVIRONMENT
<  AWX 2.0.1-225-g895a567ed  >
* AWX install method: docker-compose locally
* Ansible version:  2.7.0
* Operating System: fedora
* Web Browser: chrome

##### STEPS TO REPRODUCE
Create an organization, and create a team and a user associated with that org.
Go to the Teams tab > the team you created > Permissions tab to open a modal.
Click the "organizations" tab in the permissions modal and try to assign org-wide permissions for something on the team. 
##### EXPECTED RESULTS
Permissions are assigned.
##### ACTUAL RESULTS
Nothing happens.

cc @jakemcdermott

**Comment by dsesami** [comment]
I noticed this also occurs on the Organization Tab -> Permissions modal -> Teams tab in the permissions modal, but it ONLY does it when you try to assign Admin access.

**Comment by jakemcdermott** [comment]
Turns out organization `admin_role` and `member_role` are disallowed for teams by the API and will result in a 400 if an attempt is made. This is why there is no visual indication of a successful role assignment when the steps in this issue are followed. 

I've created a PR here which removes these items from the list of available roles: https://github.com/ansible/awx/pull/2587

I believe the 400 is expected behavior, but I'll confirm in PR review.
